tools/gulptasks/dist-release.js

```diff
@@ -307,6 +321,11 @@ async function release() {
     await removeFilesInFolder(pathToDistRepo, keepFiles);
     log.message('Successfully removed content of ' + pathToDistRepo);
 
+    // Rerun compile with sourceMaps disabled
+    await scriptsCompile(void 0, {
+        skipSourceMaps: true
+    });
+
```

Comment:
Why is a recompile necessary?

Reply:
I assumed we still want the sourceMaps on code.highcharts.com. Otherwise we can just set it false in swc